src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -144,17 +140,13 @@ const CharacterForm = ({ onClose }) => {
     })
   }
 
-  const isEdit = useMemo(() => {
-    return params && params[0] === formModeEnum.edit
-  }, [params])
-
   return (
     <>
       <Container
         className={classes.title}
         color="text.disabled"
       >
-        {isEdit ? `Edit Character - ${params[1]}` : "Add New Character"}
+        {isEdit ? `Edit Character - ${character.name}` : "Add New Character"}
```

Comment:
This can potentially crash your app. Yo are getting the `character` from redux using an id obtained from the URL, a character with the id might not exist, how can we handle this?